examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    bottom_segments = list(filter(lambda name: re.match(r"^\w+\s-\s\w+\s-\s\w+$", name) is not None, df.columns))
</pre><p>Too complicated, may be:</p><pre class="ql-syntax" spellcheck="false">len(name.split("-"))==2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>